sdk/cosmos/azure-cosmos/azure/cosmos/aio/_cosmos_client.py

```diff
@@ -310,18 +339,49 @@ async def create_database(
         _set_throughput_options(offer=offer_throughput, request_options=request_options)
 
         result = await self.client_connection.CreateDatabase(database={"id": id}, options=request_options, **kwargs)
-        return DatabaseProxy(self.client_connection, id=result["id"], properties=result)
+        if not return_headers:
+            return DatabaseProxy(self.client_connection, id=result["id"], properties=result, header=result.get_response_headers())
```

Comment:
Why are we still passing in `header=result.get_response_headers()` into the `DatabaseProxy`? We're no longer using this right?